server/modules/apicache/apicache.js

```diff
@@ -485,7 +485,7 @@ function ApiCache() {
         }
 
         if (typeof duration === "string") {
-            let split = duration.match(/^([\d\.,]+)\s?(\w+)$/);
+            let split = duration.match(/^([\d\.,]+)\s?((?:(?!\d)\w)+)$/);
```

Comment:
please see my previous comment
https://github.com/louislam/uptime-kuma/security/advisories/GHSA-5rg9-2hjc-q924#advisory-comment-118324

```suggestion
            let split = duration.match(/^([\d\.,]+)\s?([a-zA-Z]+)$/);
```